white-paper/comparison.html

```diff
@@ -319,7 +319,7 @@ <h1>Comparison of APIs in MiniApps, W3C specs, and PWAs</h1>
           <td class="c6">
             <details>
               <summary>Yes</summary>
-              Can be used within Alibaba's Native Apps including Alipay, Taobao, DingDing, Gaode, ELE.me
+              Can be used within Alibaba's native apps including Alipay, Taobao, DingDing, Gaode, ELE.me

```

Comment:
```suggestion
              Can be used within Alibaba's native apps including Alipay, Taobao, DingTalk, Gaode, ELE.me

```

[DingTalk](https://en.wikipedia.org/wiki/DingTalk) should be the right name.